autogpt_platform/autogpt_libs/autogpt_libs/supabase_integration_credentials_store/store.py

```diff
@@ -1,10 +1,14 @@
 import secrets
 from datetime import datetime, timedelta, timezone
+from typing import cast
+from backend.data import db
+from prisma import Json, Prisma
+from prisma.models import User
```

Comment:
Does the marketplace consume this class too? if not, can we make all the prisma query code lives under backend/data folder? That's where all the async function queries are living.

Reply:
TBH I have no idea why this is in libs

@Pwuts any input

Reply:
Not in scope of pr